templates/modules/b2b-account/quotes/edit-quote/quote-details.hypr.live

```diff
@@ -48,7 +48,7 @@
             </dt>
             <dd>
               {% if model.auditInfo.createDate %}
-                {{ model.auditInfo.createDate|date("m/d/Y") }}
+                <span data-date-local>{{ model.auditInfo.createDate }}</span>
```

Comment:
can we use the method here like
{{model.convertLocalDate(model.auditInfo.createDate)}}

if possible to do the above changes, update all references on the edit and view-quote page

Reply:
I tried the same but getting model.auditInfo.createDate as 'undefined' because on page load browser DOM is not ready, so that's why I added span and take value.